src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -206,6 +206,42 @@ def test_azconfig_kv(self, resource_group, location):
                          self.check('[0].value', updated_entry_value),
                          self.check('[0].label', updated_label)])
 
+        # KeyVault reference tests
```

Comment:
You can consider separate the test_azconfig_kv to several parts. In this way, you don't need to re-recording the unrelated tests and only focus on the tests related to your new features.

Reply:
In this case, it would be better to add the new feature tests in the same scenario test since I have to reuse/update some of the pre-existing resources created in the commands above. 

---
In reply to: [361830867](https://github.com/Azure/azure-cli/pull/11571#discussion_r361830867) [](ancestors = 361830867)